murano.yaml

```diff
@@ -34,6 +25,14 @@ options:
   merge: true, # To keep all existing items from solution that are not present in this template
   safeNamespace: vendor # Indicates the items with this prefix will not get remove or updated upon update
   # This will let each solution user to add custom files & behaviors surviving template verion update.
+  safeConfig:
```

Comment:
now I'm confused, it is `safeConfig` or `safeConfigs`? because in `README.md` is `safeConfigs`, here is `safeConfig`...

Reply:
```suggestion
  safeConfigs:
```